packages/eas-cli/src/submissions/ios/IosSubmitCommand.ts

```diff
@@ -62,15 +60,9 @@ class IosSubmitCommand {
   }
 
   private resolveAppSpecificPasswordSource(): Result<AppSpecificPasswordSource> {
-    const { appleAppSpecificPassword } = this.ctx.commandFlags;
     const { EXPO_APPLE_APP_SPECIFIC_PASSWORD } = process.env;
```

Comment:
Maybe use https://github.com/expo/eas-cli/blob/main/packages/eas-cli/package.json#L30 as it's already in deps.